app/config.js

```diff
@@ -9,5 +9,8 @@ module.exports = {
   port: process.env.PORT || 3000,
 
   // API_HOST
-  apiHost: process.env.API_HOST || 'http://localhost:8080/api/v1',
+  apiHost: process.env.API_HOST || 'http://localhost:8080',
+
+  // BLOBSTORE_HOST
+  blobstoreHost: process.env.BLOBSTORE_HOST || 'https://gpitfuturesdevsa.blob.core.windows.net',
```

Comment:
Maybe in future we wouldn't have a default & the app would validate the application configuration on start up?